plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintCheckTask.kt

```diff
@@ -57,6 +57,8 @@ open class KtlintCheckTask @Inject constructor(
     internal val outputToConsole: Property<Boolean> = objectFactory.property()
     @get:Console
     internal val coloredOutput: Property<Boolean> = objectFactory.property()
+    @get:Internal
```

Comment:
Not so sure if `Internal` here is good choice, as it exclude sources in task action.